templates/interfaces_nat.j2

```diff
@@ -12,3 +12,5 @@ iface lxcbr0 inet static
     # add checksum so that dhclient does not complain.
     # udp packets staying on the same host never have a checksum filled else
     post-up iptables -A POSTROUTING -t mangle -p udp --dport bootpc -s {{ lxcbr0_ip }}/{{ lxcbr0_netmask }} -j CHECKSUM --checksum-fill
+    post-up dnsmasq --pid-file=/var/run/lxc-dnsmasq.$IFACE.pid --conf-file=/etc/dnsmasq.$IFACE.conf
+    post-down kill $(cat /var/run/lxc-dnsmasq.$IFACE.pid) || true
```

Comment:
What do you think about:
```
{% if lxc_stop_dnsmasq_with_network %}
post-up dnsmasq --pid-file=/var/run/lxc-dnsmasq.$IFACE.pid --conf-file=/etc/dnsmasq.$IFACE.conf
post-down kill $(cat /var/run/lxc-dnsmasq.$IFACE.pid) || true
{% endif %}
```

May be we should add something like lxc_manage_dnsmasq and if it's true it will do it. However, if it false, it's not installed and configured as well. This is in fact a dependency.